types/loadable__component/index.d.ts

```diff
@@ -73,6 +86,14 @@ declare function loadableFunc<Props>(
     options?: OptionsWithoutResolver<Props>,
 ): LoadableComponent<Props>;
 
+declare function loadableFunc<
+    Component extends React.ComponentClass<any>,
+    Props extends React.ComponentProps<Component>
```

Comment:
Same here, I don't think the `Component` type parameter is necessary, `Props` is enough.